csrc/ir/nodes.cpp

```diff
@@ -1334,6 +1334,44 @@ std::string ReductionOp::toInlineString(int indent_size) const {
   NVF_CHECK(false, "Tensor op can not be printed inline");
 }
 
+std::vector<PolymorphicValue> ReductionOp::evaluate(
+    const ExpressionEvaluator& ee,
+    const std::vector<PolymorphicValue>& inputs) const {
+  const auto& input = inputs.at(0).as<at::Tensor>();
+  const auto output = out()->as<TensorView>();
+
+  NVF_ERROR(output->getRFactorDomain().size() == 0, 
```

Comment:
Should we use this? https://github.com/NVIDIA/Fuser/blob/6d4abc04dc4939b10e4076d4f70101a818084d26/csrc/ir/interface_nodes.h#L156